src/helpers/shifts.helper.ts

```diff
@@ -48,6 +48,21 @@ export class ShiftHelper {
     return duration === 0 ? 24 : duration;
   }
 
+  public static requiredFreeTimeAfterShift(shift: Shift): number {
+    if (this.shiftCodeToWorkTime(shift) < 9) return 11;
+    if (this.shiftCodeToWorkTime(shift) > 12) return 24;
+    return 16;
+  }
```

Comment:
@prenc 
Mi się wydaje że ta informacja powinna przychodzić z servera, bo wtedy to co pokazujemy tutaj i to uwzgłędniamy na serwerze może się rozjechać